darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -514,17 +514,28 @@ def _init_model(self, trainer: Optional[pl.Trainer] = None) -> None:
             )
 
     def _setup_trainer(
-        self, trainer: Optional[pl.Trainer], verbose: bool, epochs: int = 0
+        self,
+        trainer: Optional[pl.Trainer],
+        verbose: Optional[bool] = None,
+        epochs: int = 0,
     ) -> None:
         """Sets up the PyTorch-Lightning trainer for training or prediction."""
-
-        self.trainer_params["enable_model_summary"] = (
-            verbose if self.model.epochs_trained == 0 else False
-        )
-        self.trainer_params["enable_progress_bar"] = verbose
+        trainer_params = {key: val for key, val in self.trainer_params.items()}
+        if verbose is not None:
+            raise_deprecation_warning(
```

Comment:
Could this be time to phase out `verbose` in next release maybe?

Reply:
I thought about this and personally I actually like the flexibility and simplicity of setting verbose at trainnig/inference time. 

What again was the motivation behind removing it?

Reply:
Actually you're right, I also kind of like it. Perhaps we could un-deprecate it then? :D

Reply:
Sounds good :)